File: src/main/java/com/shopping/shopping_site_backend/infra/constant/enumconstant/OrderStatus.java
package com.shopping.shopping_site_backend.infra.constant.enumconstant;

public enum OrderStatus {
  PENDING("Pending"),
  CONFIRMED("Confirmed"), // 已确认
  SHIPPED("Shipped"),
  DELIVERED("Delivered"),
  CANCELED("Canceled"),
  REFUNDED("Refunded"); // 已退款;
  private String description;

  OrderStatus(String description) {
    this.description = description;
  }
}

File: src/main/java/com/shopping/shopping_site_backend/infra/constant/enum/OrderStatus.kt
package com.shopping.shopping_site_backend.infra.constant.enum

enum class OrderStatus(val status: String) {
    PENDING("Pending"),  // 待处理
    CONFIRMED("Confirmed"),  // 已确认
    SHIPPED("Shipped"),  // 已发货
    DELIVERED("Delivered"),  // 已送达
    CANCELED("Canceled") // 已取消
}
